bokeh/models/tools.py

```diff
@@ -1348,6 +1353,11 @@ class PolyDrawTool(EditTool, Drag, Tap):
     drag = Bool(default=True, help="""
     Enables dragging of existing patches and multi-lines on pan events.""")
 
+    overwrite = Bool(default=False, help="""
+    Whether the tool should overwrite the existing box, if there is one,
+    or whether it should add a new box on every draw event (the default).
+    """)
```

Comment:
This one isn't a box, so "existing Poly"?  Or "existing object" (here and above)?

Reply:
Right, should be polygon/object here will fix when I switch to the ``num_objects`` suggestion.